cmd/commands/server/options.go

```diff
@@ -104,6 +106,13 @@ func ParseArguments(args []string) (options CommandOptions, err error) {
 		"Address (URL form) of ipify service",
 	)
 
+	flags.StringVar(
+		&options.Protocol,
+		"protocol",
```

Comment:
I would go with configuring style like this:
```
mysterium_server 
  --identity=..
  --openvpn.binary=openvpn
  --openvpn.proto=tcp
```

Reply:
Absolutely, lets look at options as variables. "protocol" is global and doesn't say anything about context it is used. openvpn.protocol cleary states that this option is for openvpn and has something to do with openvpn connect type

Reply:
We might need to have it global, bus since we don't have other transports yet, will do as you suggest.